src/database/models.ts

```diff
@@ -67,31 +76,39 @@ export default class Models {
     static PublicEvent: mongoose.Model<PublicEvent> = undefined!;
     static EventMetadata: mongoose.Model<EventMetadata> = undefined!;
     static EventAttendance: mongoose.Model<EventAttendance> = undefined!;
+    static EventFollowing: mongoose.Model<EventFollowers> = undefined!;
     // Newsletter
     static NewsletterSubscription: mongoose.Model<NewsletterSubscription> = undefined!;
     // Registration
     static RegistrationInfo: mongoose.Model<RegistrationInfo> = undefined!;
     static RegistrationApplication: mongoose.Model<RegistrationApplication> = undefined!;
+    //Shop
+    static ShopItem: mongoose.Model<ShopItem> = undefined!;
+    static ShopQuantity: mongoose.Model<ShopQuantity> = undefined!;
     // User
     static UserInfo: mongoose.Model<UserInfo> = undefined!;
     static UserAttendance: mongoose.Model<UserAttendance> = undefined!;
 
     static initialize(): void {
         this.AttendeeMetadata = getModel(AttendeeMetadata, Database.ATTENDEE, AttendeeCollection.METADATA);
         this.AttendeeProfile = getModel(AttendeeProfile, Database.ATTENDEE, AttendeeCollection.PROFILE);
+        this.AttendeeFollowing = getModel(AttendeeFollowing, Database.ATTENDEE, AttendeeCollection.FOLLOWING);
         this.AuthInfo = getModel(AuthInfo, Database.AUTH, AuthCollection.INFO);
         this.AdmissionDecision = getModel(AdmissionDecision, Database.ADMISSION, AdmissionCollection.DECISION);
         this.StaffEvent = getModel(StaffEvent, Database.EVENT, EventCollection.STAFF_EVENTS);
         this.PublicEvent = getModel(PublicEvent, Database.EVENT, EventCollection.PUBLIC_EVENTS);
         this.EventMetadata = getModel(EventMetadata, Database.EVENT, EventCollection.METADATA);
         this.EventAttendance = getModel(EventAttendance, Database.EVENT, EventCollection.ATTENDANCE);
+        this.EventFollowing = getModel(EventFollowers, Database.EVENT, EventCollection.FOLLOWERS);
```

Comment:
```suggestion
        this.EventFollowers = getModel(EventFollowers, Database.EVENT, EventCollection.FOLLOWERS);
```

and change everywhere else